.github/workflows/build_tests.yml

```diff
@@ -422,7 +422,7 @@ jobs:
         run: |
           bazel build --compilation_mode=opt -- //:oppia_alpha_kitkat
 
-      # Note that caching only works on non-forks.
+      # Note that caching only works on non-forks. 4419: should remove this?
```

Comment:
Yes, should be removed, because this workflow will start failing now that the kenya build has been removed.